tests/components/unifi/test_controller.py

```diff
@@ -147,31 +159,22 @@ async def test_controller_setup(hass):
         controller = await setup_unifi_integration(hass)
 
     entry = controller.config_entry
-    assert len(forward_entry_setup.mock_calls) == len(
-        unifi.controller.SUPPORTED_PLATFORMS
-    )
-    assert forward_entry_setup.mock_calls[0][1] == (entry, "device_tracker")
-    assert forward_entry_setup.mock_calls[1][1] == (entry, "sensor")
-    assert forward_entry_setup.mock_calls[2][1] == (entry, "switch")
+    assert len(forward_entry_setup.mock_calls) == len(SUPPORTED_PLATFORMS)
+    assert forward_entry_setup.mock_calls[0][1] == (entry, TRACKER_DOMAIN)
+    assert forward_entry_setup.mock_calls[1][1] == (entry, SENSOR_DOMAIN)
+    assert forward_entry_setup.mock_calls[2][1] == (entry, SWITCH_DOMAIN)
 
     assert controller.host == CONTROLLER_DATA[CONF_HOST]
     assert controller.site == CONTROLLER_DATA[CONF_SITE_ID]
     assert controller.site_name in SITES
     assert controller.site_role == SITES[controller.site_name]["role"]
 
-    assert (
-        controller.option_allow_bandwidth_sensors
-        == unifi.const.DEFAULT_ALLOW_BANDWIDTH_SENSORS
-    )
+    assert controller.option_allow_bandwidth_sensors == DEFAULT_ALLOW_BANDWIDTH_SENSORS
     assert isinstance(controller.option_block_clients, list)
-    assert controller.option_track_clients == unifi.const.DEFAULT_TRACK_CLIENTS
-    assert controller.option_track_devices == unifi.const.DEFAULT_TRACK_DEVICES
-    assert (
-        controller.option_track_wired_clients == unifi.const.DEFAULT_TRACK_WIRED_CLIENTS
-    )
-    assert controller.option_detection_time == timedelta(
-        seconds=unifi.const.DEFAULT_DETECTION_TIME
-    )
+    assert controller.option_track_clients == DEFAULT_TRACK_CLIENTS
+    assert controller.option_track_devices == DEFAULT_TRACK_DEVICES
+    assert controller.option_track_wired_clients == DEFAULT_TRACK_WIRED_CLIENTS
+    assert controller.option_detection_time == timedelta(seconds=DEFAULT_DETECTION_TIME)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5403678)